<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>DWD</class>
  <widget class="qMRMLWidget" name="DWD">
    <layout class="QVBoxLayout">
      <item>
        <widget class="ctkCollapsibleButton">
          <property name="text">
            <string>Training</string>
          </property>
          <layout class="QFormLayout">
            <item row="0" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Training Dataset</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="ctkPathLineEdit" name="path_train">
              </widget>
            </item>
            <item row="1" column="0" colspan="2">
              <widget class="QCheckBox" name="chk_autoTune">
                <property name="text">
                  <string>Automatic Tuning</string>
                </property>
                <property name="checked">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Tuning Parameter (C)</string>
                </property>
              </widget>
            </item>
            <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="spn_tuningC">
                <property name="minimum">
                  <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                  <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                  <double>0.010000000000000</double>
                </property>
                <property name="value">
                  <double>0.010000000000000</double>
                </property>
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Output Model</string>
                </property>
              </widget>
            </item>
            <item row="3" column="1">
              <widget class="qMRMLNodeComboBox" name="outputSelector">
                <property name="nodeTypes">
                  <stringlist>
                    <string>vtkMRMLModelNode</string>
                  </stringlist>
                </property>
                <property name="showChildNodeTypes">
                  <bool>false</bool>
                </property>
                <property name="addEnabled">
                  <bool>true</bool>
                </property>
                <property name="removeEnabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="0" colspan="2">
              <widget class="QPushButton">
                <property name="text">
                  <string>Train Classifier</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton">
          <property name="text">
            <string>Testing</string>
          </property>
          <layout class="QFormLayout">
            <item row="0" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Test Dataset</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="ctkPathLineEdit" name="path_test"/>
            </item>
            <item row="1" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Test Results</string>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="ctkPathLineEdit" name="path_testResults"/>
            </item>
            <item row="2" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Testing Statistics</string>
                </property>
              </widget>
            </item>
            <item row="3" column="0" colspan="2">
              <widget class="QTableWidget" name="tbl_stats">
              </widget>
            </item>
            <item row="4" column="0" colspan="2">
              <widget class="QPushButton">
                <property name="text">
                  <string>Test Classifier</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton">
          <property name="text">
            <string>Analysis</string>
          </property>
          <layout class="QFormLayout">
            <item row="0" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Input Dataset</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="ctkPathLineEdit" name="path_input"/>
            </item>
            <item row="1" column="0" colspan="2">
              <widget class="QPushButton">
                <property name="text">
                  <string>Compute Projected Mean Shape</string>
                </property>
              </widget>
            </item>
            <item row="2" column="0" colspan="2">
              <widget class="QPushButton">
                <property name="text">
                  <string>Correlate with Existing Metrics</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>40</height>
            </size>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>ctkCollapsibleButton</class>
      <extends>QWidget</extends>
      <header>ctkCollapsibleButton.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkPathLineEdit</class>
      <extends>QWidget</extends>
      <header>ctkPathLineEdit.h</header>
    </customwidget>
    <customwidget>
      <class>ctkSliderWidget</class>
      <extends>QWidget</extends>
      <header>ctkSliderWidget.h</header>
    </customwidget>
    <customwidget>
      <class>qMRMLNodeComboBox</class>
      <extends>QWidget</extends>
      <header>qMRMLNodeComboBox.h</header>
    </customwidget>
    <customwidget>
      <class>qMRMLWidget</class>
      <extends>QWidget</extends>
      <header>qMRMLWidget.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections>
    <!--    <connection>-->
    <!--      <sender>DWD</sender>-->
    <!--      <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>-->
    <!--      <receiver>inputSelector</receiver>-->
    <!--      <slot>setMRMLScene(vtkMRMLScene*)</slot>-->
    <!--    </connection>-->
    <connection>
      <sender>DWD</sender>
      <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
      <receiver>outputSelector</receiver>
      <slot>setMRMLScene(vtkMRMLScene*)</slot>
    </connection>
    <!--    <connection>-->
    <!--      <sender>DWD</sender>-->
    <!--      <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>-->
    <!--      <receiver>invertedOutputSelector</receiver>-->
    <!--      <slot>setMRMLScene(vtkMRMLScene*)</slot>-->
    <!--    </connection>-->
  </connections>
</ui>
